engine/src/main/java/org/terasology/rendering/dag/stateChanges/EnableDepthTest.java

```diff
@@ -20,9 +20,14 @@
 import static org.lwjgl.opengl.GL11.GL_DEPTH_TEST;
 
 /**
- * Instances of this enable OpenGL's depth test, which allows fragments that fail the test to be discarded.
+ * Enables depth testing in OpenGL pipeline.
+ * This allows the processing of fragments that are hidden behind another fragment, and would normally fail the test and therefore be discarded.
  *
- * This StateChange is used to reset the effects of DisableDepthTest.
+ * OpenGL Default: DisableDepthTest
+ * Terasology Default: EnableDepthTest
+ *
+ * Type: Set
+ * Corresponding Reset: EnableDepthTest
```

Comment:
Similarly to a previous comment, this is actually a reset class given that Terasology's default is for the depth test to be enabled.